examples/platform/silabs/SiWx917/BaseApplication.cpp

```diff
@@ -58,7 +61,7 @@
  * Defines and Constants
  *********************************************************/
 
-#define FACTORY_RESET_TRIGGER_TIMEOUT 3000
+#define FACTORY_RESET_TRIGGER_TIMEOUT 7000
```

Comment:
Why Change the Timeout ? Is there a specific reason why the 917 should be different on that front ?

Reply:
No I added both Trigger timeout and cancel timeout plus 1 sec of wait before checking the status.